### ISSUE TITLE
Completed calendar events no longer show a style change

### ISSUE THREAD

**Issue by bandtank** [espocrm/espocrm#2776]
When setting the status of an event to `Held` or `Not Held`, the style no longer changes after upgrading from 7.4.9 to 7.5.3.
- I expect an event with a status of `Held` to become darker.
- I expect an event with a status of `Not Held` to become darker and for the text to have a strikethrough line.

Neither style change occurs at this point and I'm not sure why.

**Comment by bandtank** [comment]
For anyone else who stumbles on this, I had edited the application's files to have custom completed and canceled status indicators. I noticed in the 7.5.3 update files that the statuses are now pulled from the entity's metadata, which means the values can be set in the entity's settings. Edit your entities and add the statuses to the new multi-enum fields:

![image](https://github.com/espocrm/espocrm/assets/1170415/e80db7ec-8a05-4c65-9c55-0206d83f3201)

This is a great upgrade. Thanks, Yuri.
